### PR TITLE
feat(be): ADP-2517: table improvements

### PR DESCRIPTION
**Pull request by IAleks94**
image

### CODE REVIEW

File: src/services/issues.service.ts [hunk 1]
```diff
@@ -39,14 +40,25 @@ export class IssuesService {
...
     issues.forEach((issue, index) => {
-      table.push([index + 1, issue.name, issue.severity, issue.dependencyName, issue.dependencyInstalledVersion]);
+      table.push(this.transformIssueToRow(issue, index));
     });
 
     console.log(table.toString());
   }
+
+  transformIssueToRow(issue: IIssueResponse, index: number) {
+    const severity = capitalize(issue.severity);
```
Comment (noamilshtein) [inline src/services/issues.service.ts]: why do we need to use lodash for capitalize? not a complex task

Comment (IAleks94) [reply src/services/issues.service.ts]: we already have the lodash package, I prefer to use it instead of creating the same functions

Comment (IAleks94) [reply src/services/issues.service.ts]: image

Comment (IAleks94) [reply src/services/issues.service.ts]: Lodash is already used in libs that we use, if we don't use it directly it is in our bundle anyway

**Review by noamilshtein — APPROVED** [review]

**Pull request merged**